core/storage/question/gae_models.py

```diff
@@ -316,25 +316,58 @@ def get_question_skill_links_by_skill_ids(
         ) * question_count
 
         if not start_cursor == '':
-            cursor = datastore_query.Cursor(urlsafe=start_cursor)
-            question_skill_link_models, next_cursor, more = cls.query(
-                cls.skill_id.IN(skill_ids)
-                # Order by cls.key is needed alongside cls.last_updated so as to
-                # resolve conflicts, if any.
-                # Reference SO link: https://stackoverflow.com/q/12449197
-            ).order(-cls.last_updated, cls.key).fetch_page(
-                question_skill_count,
-                start_cursor=cursor
-            )
+            direction = start_cursor[-1]
+            random_id = unicode(start_cursor[-13:-1]) #pylint: disable=unicode-builtin
```

Comment:
You should not be doing unicode conversions (in general if you have to use a pylint pragma, something is probably going wrong).

Please explain the rationale (but I think you might actually need to find a different approach here).

Reply:
I used unicode because the question id was in unicode and I thought that filtering required unicode conversion but it worrked without conversion. So i removed it.